### PR TITLE
Add KeyboardFocus page to Table of Contents

### PR DESCRIPTION
**Pull request by Saadnajmi**
Guess I forgot to do this 🤷🏾

### CODE REVIEW

**Comment by Saadnajmi** [comment]
@anandrajeswaran @harrieshin

**Review by harrieshin — APPROVED** [review]

**Comment by Saadnajmi** [comment]
Can't merge this, so it would be great if someone with write permissions did!

**Pull request merged**
